### DISCUSSION TITLE
Just installed, love it, but it's missing some things that would make it great.

### DISCUSSION THREAD

**Discussion by Savasdotexe** [SergiX44/XBackBone#358] [Q&A]
Single user mode - Option to do away with 'User Code' so instead of example.com/xxxxx/file.png, just have it as example.com/file.png or have it as something the user can change, I guess this is similar to simply requesting the ability to change user code and also do away with it.

Upload without logging in - This could be same as being a user, but with a lot more limits, like size, file, etc.

**Comment by sergix44** [comment]
Those features was actually asked in the past:
https://github.com/SergiX44/XBackBone/issues/36
https://github.com/SergiX44/XBackBone/issues/119
https://github.com/SergiX44/XBackBone/issues/297

A single use mode is not actually possible because was thought to be multi-user by design. To add the ability to make it single user would require a lot of breaking changes in the v3 codebase, same thing about the public uploads.

I was thinkering about making a new v4 app, so those may be added in the future.
